src/app/src/components/FacilityListItems.jsx

```diff
@@ -170,10 +180,22 @@ class FacilityListItems extends Component {
                 onClick={downloadCSV}
                 disabled={downloadingCSV}
             >
-                Download CSV
+                Download Formatted CSV
```

Comment:
Not directly related to the new functionality, but since we accept and encourage using Excel files we should consider changing "CSV" here to "File"